litestar/contrib/sqlalchemy/base.py

```diff
@@ -42,7 +43,7 @@
 UUIDBaseT = TypeVar("UUIDBaseT", bound="UUIDBase")
 BigIntBaseT = TypeVar("BigIntBaseT", bound="BigIntBase")
 
-convention = {
+convention: NamingSchemaParameter = {
```

Comment:
Oh, that's good fix. Didn't know about this secret type alias when I added the `type: ignore` :grimacing: